<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRcodeClass</class>
 <widget class="QDialog" name="QRcodeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>二维码生成小工具 - DQN</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>1</y>
     <width>541</width>
     <height>602</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayoutMain">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="LayoutHead">
      <item>
       <widget class="QLineEdit" name="edit_content">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_generate">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>生成</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget_colors" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_colors{background:#fff;}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_render">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_render{
background:#fff;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QRcode.qrc"/>
 </resources>
 <connections/>
</ui>
